test/Signature.ts

```diff
@@ -4403,40 +4116,40 @@ describe('Signature', function () {
         },
       ]);
 
-      const paymentElements = await this.driver.findElements(
-        By.css('[data-testid="invokeScriptPaymentItem"]')
-      );
-
       const actualPayments = await Promise.all(
-        paymentElements.map(el => el.getText())
+        await invokeScript.invokeScriptPaymentItems.map(async it =>
+          it.getText()
+        )
       );
-
-      expect(actualPayments).to.deep.equal([
+      expect(actualPayments).toStrictEqual([
         '0.00000001 WAVES',
         '1 NonScriptToken',
       ]);
 
-      expect(
-        await invokeScript
-          .findElement(By.css('[data-testid="txFee"]'))
-          .getText()
-      ).to.equal('0.005 WAVES');
+      expect(await invokeScript.fee).toHaveText('0.005 WAVES');
 
-      await rejectMessage.call(this);
-      await closeMessage.call(this);
+      await CommonTransaction.rejectButton.click();
+      await FinalTransactionScreen.root.waitForDisplayed();
+      await FinalTransactionScreen.closeButton.click();
     });
 
     it('Approved', async function () {
-      await performSignTransactionPackage.call(this, PACKAGE, 'Test package');
+      await performSignTransactionPackage(
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        PACKAGE as any,
+        'Test package'
+      );
 
-      await approveMessage.call(this);
-      await closeMessage.call(this);
+      await CommonTransaction.approveButton.click();
+      await FinalTransactionScreen.root.waitForDisplayed();
+      await FinalTransactionScreen.closeButton.click();
 
-      const approveResult = await this.driver.executeScript<string[]>(
-        () => window.result
+      await browser.switchToWindow(tabOrigin);
+      const approveResult = await browser.execute(
+        () => window.result as string[]
```

Comment:
Why not use generic instead of type cast here?

Reply:
done